### ISSUE TITLE
relay: paginating backward with cursors does not work

### ISSUE THREAD

**Issue by majodev** [mickhansen/graphql-sequelize#430]
Hi!

See this [branch](https://github.com/majodev/graphql-sequelize/commit/b22e8e5dc456f18d1d4bf16f990e8b7acd0bc47b) for a reproduceable test.

TLDR:
1. Get first 3 items
2. Use `endCursor` from 1. response in `after` arg to get next `first` 3 items
3. Use `startCursor` from 2. response in `before` to get `first` 3 items again (fails)

It seems like we need to subtract the computed offset from on the computed cursor [here](https://github.com/mickhansen/graphql-sequelize/blob/master/src/relay.js#L268) and also adapt the [resolveEdge handling](https://github.com/mickhansen/graphql-sequelize/blob/master/src/relay.js#L201) and possibly adapt the [hasNextPage/hasPreviousPage handling](https://github.com/mickhansen/graphql-sequelize/blob/master/src/relay.js#L324).

Quite a lot, not sure if I'm 100% right, can someone please confirm this?

If this handling is actually really expect in relay, how to you guys paginate forward and backwards correctly?

**Comment by majodev** [comment]
Uups, nevermind. This is supposed to work like this!

Lets keep this issue here for [reference](https://facebook.github.io/relay/graphql/connections.htm#sec-Forward-pagination-arguments). You need to choose if you want to go forward or backward...
